File: src/widgets/measurementtable.cpp
#include "measurementtable.h"
#include "ui_measurementtable.h"

MeasurementTable::MeasurementTable(QWidget *parent) :
    QWidget(parent),
    ui(new Ui::MeasurementTable)
{
    ui->setupUi(this);
}

MeasurementTable::~MeasurementTable()
{
    delete ui;
}

File: dcs/client/ui/measurement_table.py
from PySide6 import QtWidgets

from .measurement_table_ui import Ui_MeasurementTable

class MeasurementTableWidget(QtWidgets.QWidget, Ui_MeasurementTable):
    def __init__(self, parent=None):
        super().__init__(parent)
        self.setupUi(self)